client/components/reducers/reducers.js

```diff
@@ -0,0 +1,14 @@
+// const state = {
```

Comment:
Why is this file commented? Will store work without reducer?

Reply:
Will be elaborated today.